app/helpers/discussions_helper.rb

```diff
@@ -113,7 +113,7 @@ def discussions_statuses
   end
 
   def discussions_languages(discussions)
-    @languages ||= discussions.map { |it| it.language.name }.uniq
+    @languages ||= discussions.map { |it| it.exercise.language.name }.uniq
```

Comment:
This one avoids querying for already obtained language. See https://github.com/mumuki/mumuki-domain/pull/114/files#diff-955c51844f31b51f452cbca650a8bf48R21

Reply:
So that means that if you use `includes`, it pre-fetch direct relations but not-through relations? 

It makes sense, but it make me wonder how brittle this optimization is.

Reply:
Perhaps it could be less error prone to replace the through relation in `Discussion`, and instead add an explicit delegate method with a comment, either with `delegate to:` or with a plain standard method